net/tstun/wrap.go

```diff
@@ -144,6 +144,11 @@ type Wrapper struct {
 	// OnTSMPPongReceived, if non-nil, is called whenever a TSMP pong arrives.
 	OnTSMPPongReceived func(packet.TSMPPongReply)
 
+	// OnICMPEchoReceived, if non-nil, is called whenever a ICMP echo response
```

Comment:
Put Response in the name too? Otherwise I think we'll be confused every time we read it.